doc/release-notes-1.11.rst

```diff
@@ -55,10 +55,33 @@ Boards
   * Includes a template for hardware models
   * Adds support for console and logging
   * Interrupts and timers are simulated in several different configurations
+* Added support for the following ARM boards:
```

Comment:
Hmmm... looks like some stray files getting into this PR?

Reply:
Yeap, I should have notices that the base branch was changed. Pls, don't review until i figure out how to exclude these.